app/views/additional_document_validation_requests/_additional_document_validation_requests_table.html.erb

```diff
@@ -21,7 +21,7 @@
         </td>
         <td class="govuk-table__cell govuk-!-width-one-sixth">
           <% if validation_request.open_or_pending? %>
-            <% if @planning_application.invalidated? %>
+            <% if @planning_application.invalidated? || validation_request.post_validation? %>
```

Comment:
Huh looks like I added a method `can_cancel?` but it's not being used anymore. Wonder if it's worth adding this logic in there and reviving it!

I think I may have overlooked this too when adding this same if statement for other validation request types so i'm happy to just create a separate PR to update all of these if you prefer that ?

https://github.com/unboxed/bops/blob/main/app/models/concerns/validation_request.rb#L118

Reply:
Oh nice!